lblod/frontend-embeddable-notule-editor
# GN-4516: solve svg cors issues

## Overview
This PR ensures that svgs rendered by the `au-icon` component are rendered inline, instead of requesting a symbolset file.
It does so by overriding the `au-icon` component provided by the `@appuniversum/ember-appuniversum` package.
In order to render the svgs inline, this PR adds the `ember-svg-jar` package.

This PR solves cors issues and issues when trying to correctly resolve the svg symbolset file from the appuniversum package.

This PR also removes the `svgxuse` package, which was used as a workaround for the cors issues occurring when request the svg symbolset file.

##### connected issues and PRs:
[GN-4516](https://binnenland.atlassian.net/browse/GN-4516?atlOrigin=eyJpIjoiZmE2NDE4YWQ5NDFlNDQxNTg3ODg3YmNiNjkzYTY2YzQiLCJwIjoiaiJ9)

### How to test/reproduce
- npm install
- npm start
- check if the symbols are correctly rendered
- no cors issues should appear
- when inspecting the icons using the browser-inspector, you should notice that the svg code is rendered inline

### Checks PR readiness
